type-definitions/immutable.js.flow

```diff
@@ -868,6 +868,7 @@ declare class Map<K, +V> extends KeyedCollection<K, V> mixins UpdatableInCollect
 
   size: number;
 
+  fromOwnEntries(obj: mixed): Map<K, V>;
```

Comment:
This would be `static`, and ideally the `K, V` are mapped to something on the input, see `static <K, V>(...)` above, which is the standard constructor.